account/src/main/java/bisq/account/protocol_type/LoanProtocolType.java

```diff
@@ -18,17 +18,21 @@
 package bisq.account.protocol_type;
 
 import bisq.common.util.ProtobufUtils;
+import bisq.common.util.StringUtils;
 
 public enum LoanProtocolType implements ProtocolType {
     COLLATERALIZED,
     REPUTATION;
 
+    private static final String protobufPrefix = StringUtils.capitalizeAll(LoanProtocolType.class.getSimpleName()) + "_";
```

Comment:
This can be moved to the ProtoEnum and the static method reused by ProtobufUtils

```
public interface ProtoEnum {
    ProtocolMessageEnum toProto();

    default String getProtobufPrefix() {
        return getProtobufPrefix(this.getClass());
    }

    static String getProtobufPrefix(Class<?> clazz) {
        return StringUtils.capitalizeAll(clazz.getSimpleName()) + "_";
    }
}


in ProtobufUtils.enumFromProto:
 String enumName = name.replace(ProtoEnum.getProtobufPrefix(enumType),"");
       
```

Reply:
Great suggestion. Will add it to the code.